src/style.css

```
@@ -1 +1,66 @@
 /* Styles go here */
+/* Styles go here */
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-content: center;
+}
+
+.nav {
+  margin-right: 50px;
+}
```

```suggestion
.nav {
}
```
Set horizontal paddigns to the header instead